Unable to setup on Windows 10

**My environment**
Windows 10
Git 2.12.2.windows.1
.NET Core SDK 1.0.1
(just in case, I do not have any version of Visual Studio installed)

**My issue:**
When I clone the repository and run setup.cmd, toward the end of the script execution, an error is printed

```
C:\Users\lambdakris\Code\templating\.dotnet\sdk\1.0.0-rc4-004917\Microsoft.Common.CurrentVersion.targets(1111,5): error MSB3644: The reference assemblies for framework ".NETFramework,Version=v4.5" were not found. To resolve this, install the SDK or Targeting Pack for this framework version or retarget your application to a version of the framework for which you have the SDK or Targeting Pack installed. Note that assemblies will be resolved from the Global Assembly Cache (GAC) and will be used in place of reference assemblies. Therefore your assembly may not be correctly targeted for the framework you intend. [C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Abstractions\Microsoft.TemplateEngine.Abstractions.csproj]
```

Nonetheless, the script continues and at the end prints out `dotnet new3 is ready!`.

Unfortunately, when I run `dotnet new3` I receive the error message 'No executable found matching command "dotnet-new3"'. 

**Things I tried:**
1. Considering the warning in https://github.com/dotnet/templating/wiki/Contributing#note-for-windows-users, I checked to see if I could find the location of dotnet-new3.exe, but it was nowhere to be found.
2. Since the error message mentions .NET 4.5, I went ahead and tried installing .NET 4.5.2 Dev Pack to see if this would address or change the error message printed during the execution of setup.cmd but it did not have any effect.

**Complete output of the setup.cmd script:**

```
Using build configuration "Release"...
The system cannot find the file specified.
The system cannot find the file specified.
The system cannot find the file specified.
The system cannot find the file specified.
"Calling build.ps1"
VERBOSE: dotnet-install: Get-Azure-Channel-From-Channel -Channel "rel-1.0.0"
VERBOSE: dotnet-install: Get-CLIArchitecture-From-Architecture -Architecture "<auto>"
VERBOSE: dotnet-install: Get-Machine-Architecture
VERBOSE: dotnet-install: Get-CLIArchitecture-From-Architecture -Architecture "AMD64"
VERBOSE: dotnet-install: Get-Specific-Version-From-Version -AzureFeed "https://dotnetcli.azureedge.net/dotnet" -AzureChannel "rel-1.0.0" -CLIArchitecture "x64" -Version
"Latest"
VERBOSE: dotnet-install: Get-Latest-Version-Info -AzureFeed "https://dotnetcli.azureedge.net/dotnet" -AzureChannel "rel-1.0.0" -CLIArchitecture "x64"
VERBOSE: dotnet-install: Get-Version-Info-From-Version-Text -VersionText "9a4c2c32f4a9279765c0d631f605575a080aeba3
1.0.0-rc4-004917
"
VERBOSE: dotnet-install: Get-Download-Links -AzureFeed "https://dotnetcli.azureedge.net/dotnet" -AzureChannel "rel-1.0.0" -SpecificVersion "1.0.0-rc4-004917"
-CLIArchitecture "x64"
VERBOSE: dotnet-install: Constructed payload URL: https://dotnetcli.azureedge.net/dotnet/Sdk/1.0.0-rc4-004917/dotnet-dev-win-x64.1.0.0-rc4-004917.zip
VERBOSE: dotnet-install: Resolve-Installation-Path -InstallDir "<auto>"
VERBOSE: dotnet-install: Get-User-Share-Path
VERBOSE: dotnet-install: InstallRoot: C:\Users\lambdakris\Code\templating\.dotnet\
VERBOSE: dotnet-install: Is-Dotnet-Package-Installed -InstallRoot "C:\Users\lambdakris\Code\templating\.dotnet\" -RelativePathToPackage "sdk" -SpecificVersion
"1.0.0-rc4-004917"
VERBOSE: dotnet-install: Is-Dotnet-Package-Installed: Path to a package: C:\Users\lambdakris\Code\templating\.dotnet\sdk\1.0.0-rc4-004917
VERBOSE: dotnet-install: .NET SDK installed? True
dotnet-install: .NET SDK version 1.0.0-rc4-004917 is already installed.
dotnet-install: Adding to current process PATH: "C:\Users\lambdakris\Code\templating\.dotnet\". Note: This change will not be visible if PowerShell was run as a child process.
Microsoft (R) Build Engine version 15.1.548.43366
Copyright (C) Microsoft Corporation. All rights reserved.

  939
  b387177b776b6741353b3dbc799855f509b350b1
  Restoring packages for C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Abstractions\Microsoft.TemplateEngine.Abstractions.csproj...
  Lock file has not changed. Skipping lock file write. Path: C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Abstractions\obj\project.assets.json
  Restore completed in 230.54 ms for C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Abstractions\Microsoft.TemplateEngine.Abstractions.csproj.

  NuGet Config files used:
      C:\Users\lambdakris\Code\templating\NuGet.Config
      C:\Users\lambdakris\AppData\Roaming\NuGet\NuGet.Config

  Feeds used:
      https://dotnet.myget.org/F/cli-deps/api/v3/index.json
      https://api.nuget.org/v3/index.json
  Microsoft.TemplateEngine.Abstractions -> C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Abstractions\bin\Release\netstandard1.3\Microsoft.TemplateEngine.Abstractions.dll
  Restoring packages for C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Abstractions\Microsoft.TemplateEngine.Abstractions.csproj...
  Restoring packages for C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Core.Contracts\Microsoft.TemplateEngine.Core.Contracts.csproj...
  Lock file has not changed. Skipping lock file write. Path: C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Abstractions\obj\project.assets.json
  Restore completed in 232.42 ms for C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Abstractions\Microsoft.TemplateEngine.Abstractions.csproj.
  Lock file has not changed. Skipping lock file write. Path: C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Core.Contracts\obj\project.assets.json
  Restore completed in 231.27 ms for C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Core.Contracts\Microsoft.TemplateEngine.Core.Contracts.csproj.

  NuGet Config files used:
      C:\Users\lambdakris\Code\templating\NuGet.Config
      C:\Users\lambdakris\AppData\Roaming\NuGet\NuGet.Config

  Feeds used:
      https://dotnet.myget.org/F/cli-deps/api/v3/index.json
      https://api.nuget.org/v3/index.json
  Microsoft.TemplateEngine.Core.Contracts -> C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Core.Contracts\bin\Release\netstandard1.3\Microsoft.TemplateEngine.Core.Contracts.dll
  Restoring packages for C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Abstractions\Microsoft.TemplateEngine.Abstractions.csproj...
  Restoring packages for C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Utils\Microsoft.TemplateEngine.Utils.csproj...
  Lock file has not changed. Skipping lock file write. Path: C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Abstractions\obj\project.assets.json
  Restore completed in 267.43 ms for C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Abstractions\Microsoft.TemplateEngine.Abstractions.csproj.
  Lock file has not changed. Skipping lock file write. Path: C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Utils\obj\project.assets.json
  Restore completed in 270.05 ms for C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Utils\Microsoft.TemplateEngine.Utils.csproj.

  NuGet Config files used:
      C:\Users\lambdakris\Code\templating\NuGet.Config
      C:\Users\lambdakris\AppData\Roaming\NuGet\NuGet.Config

  Feeds used:
      https://dotnet.myget.org/F/cli-deps/api/v3/index.json
      https://api.nuget.org/v3/index.json
  Microsoft.TemplateEngine.Utils -> C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Utils\bin\Release\netstandard1.3\Microsoft.TemplateEngine.Utils.dll
  Restoring packages for C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Utils\Microsoft.TemplateEngine.Utils.csproj...
  Restoring packages for C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Core.Contracts\Microsoft.TemplateEngine.Core.Contracts.csproj...
  Restoring packages for C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Abstractions\Microsoft.TemplateEngine.Abstractions.csproj...
  Restoring packages for C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Core\Microsoft.TemplateEngine.Core.csproj...
  Lock file has not changed. Skipping lock file write. Path: C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Core.Contracts\obj\project.assets.json
  Restore completed in 201.31 ms for C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Core.Contracts\Microsoft.TemplateEngine.Core.Contracts.csproj.
  Lock file has not changed. Skipping lock file write. Path: C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Abstractions\obj\project.assets.json
  Restore completed in 204.62 ms for C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Abstractions\Microsoft.TemplateEngine.Abstractions.csproj.
  Lock file has not changed. Skipping lock file write. Path: C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Utils\obj\project.assets.json
  Restore completed in 209.46 ms for C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Utils\Microsoft.TemplateEngine.Utils.csproj.
  Lock file has not changed. Skipping lock file write. Path: C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Core\obj\project.assets.json
  Restore completed in 202.12 ms for C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Core\Microsoft.TemplateEngine.Core.csproj.

  NuGet Config files used:
      C:\Users\lambdakris\Code\templating\NuGet.Config
      C:\Users\lambdakris\AppData\Roaming\NuGet\NuGet.Config

  Feeds used:
      https://dotnet.myget.org/F/cli-deps/api/v3/index.json
      https://api.nuget.org/v3/index.json
  Microsoft.TemplateEngine.Core -> C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Core\bin\Release\netstandard1.3\Microsoft.TemplateEngine.Core.dll
  Restoring packages for C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Abstractions\Microsoft.TemplateEngine.Abstractions.csproj...
  Restoring packages for C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Utils\Microsoft.TemplateEngine.Utils.csproj...
  Restoring packages for C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Orchestrator.RunnableProjects\Microsoft.TemplateEngine.Orchestrator.RunnableProjects.csproj...
  Restoring packages for C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Core.Contracts\Microsoft.TemplateEngine.Core.Contracts.csproj...
  Restoring packages for C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Core\Microsoft.TemplateEngine.Core.csproj...
  Lock file has not changed. Skipping lock file write. Path: C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Core.Contracts\obj\project.assets.json
  Restore completed in 388.18 ms for C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Core.Contracts\Microsoft.TemplateEngine.Core.Contracts.csproj.
  Lock file has not changed. Skipping lock file write. Path: C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Abstractions\obj\project.assets.json
  Restore completed in 397.12 ms for C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Abstractions\Microsoft.TemplateEngine.Abstractions.csproj.
  Lock file has not changed. Skipping lock file write. Path: C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Core\obj\project.assets.json
  Restore completed in 397.41 ms for C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Core\Microsoft.TemplateEngine.Core.csproj.
  Lock file has not changed. Skipping lock file write. Path: C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Utils\obj\project.assets.json
  Restore completed in 404.91 ms for C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Utils\Microsoft.TemplateEngine.Utils.csproj.
  Lock file has not changed. Skipping lock file write. Path: C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Orchestrator.RunnableProjects\obj\project.assets.json
  Restore completed in 450.37 ms for C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Orchestrator.RunnableProjects\Microsoft.TemplateEngine.Orchestrator.RunnableProjects.csproj.

  NuGet Config files used:
      C:\Users\lambdakris\Code\templating\NuGet.Config
      C:\Users\lambdakris\AppData\Roaming\NuGet\NuGet.Config

  Feeds used:
      https://dotnet.myget.org/F/cli-deps/api/v3/index.json
      https://api.nuget.org/v3/index.json
  Microsoft.TemplateEngine.Orchestrator.RunnableProjects -> C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Orchestrator.RunnableProjects\bin\Release\netstandard1.3\Microsoft.TemplateEngine.Orchestrator.RunnableProjects.dll
  Restoring packages for C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Abstractions\Microsoft.TemplateEngine.Abstractions.csproj...
  Lock file has not changed. Skipping lock file write. Path: C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Abstractions\obj\project.assets.json
  Restore completed in 121.64 ms for C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Abstractions\Microsoft.TemplateEngine.Abstractions.csproj.

  NuGet Config files used:
      C:\Users\lambdakris\Code\templating\NuGet.Config
      C:\Users\lambdakris\AppData\Roaming\NuGet\NuGet.Config

  Feeds used:
      https://dotnet.myget.org/F/cli-deps/api/v3/index.json
      https://api.nuget.org/v3/index.json
C:\Users\lambdakris\Code\templating\.dotnet\sdk\1.0.0-rc4-004917\Microsoft.Common.CurrentVersion.targets(1111,5): error MSB3644: The reference assemblies for framework ".NETFramework,Version=v4.5" were not found. To resolve this, install the SDK or Targeting Pack for this framework version or retarget your application to a version of the framework for which you have the SDK or Targeting Pack installed. Note that assemblies will be resolved from the Global Assembly Cache (GAC) and will be used in place of reference assemblies. Therefore your assembly may not be correctly targeted for the framework you intend. [C:\Users\lambdakris\Code\templating\src\Microsoft.TemplateEngine.Abstractions\Microsoft.TemplateEngine.Abstractions.csproj]
Artifacts built and placed.
Copying configuration for builtins...
Deleting NuGet caches...
File Not Found
File Not Found
Done.

You can now use `setup` from anywhere (in this console session) to run setup again.
You can now use `build` from anywhere (in this console session) to build dotnet-new3 in the current configuration.
You can now use `debug` from anywhere (in this console session) to change the active build configuration to DEBUG.
You can now use `release` from anywhere (in this console session) to change the active build configuration to RELEASE.

dotnet new3 is ready!
```

--- a/build.ps1
+++ b/build.ps1
@@ -6,6 +6,7 @@
 param(
     [string]$Configuration="Debug",
     [string]$Runtime="win7-x86",
+    [bool]$PerformFullFrameworkBuild=$true,
     [switch]$Help)
 
 if($Help)
@@ -13,9 +14,19 @@ if($Help)
     Write-Host "Usage: .\build.ps1 [-Configuration <CONFIGURATION>] [-Help]"
     Write-Host ""
     Write-Host "Options:"
-    Write-Host "  -Configuration <CONFIGURATION>     Build the specified Configuration (Debug or Release, default: Debug)"
-    Write-Host "  -Help                              Display this help message"
+    Write-Host "  -Configuration <CONFIGURATION>                Build the specified Configuration (Debug or Release, default: Debug)"
+    Write-Host "  -PerformFullFrameworkBuild <$true|$false>     Whether or not to build for NET45 as well as .NET Core"
+    Write-Host "  -Help                                         Display this help message"
     exit 0
+}
+
+if (!$PerformFullFrameworkBuild)
+{
+    Write-Host "Skipping NET45 build..."
+}
+else
+{
+    Write-Host "NET45 build is enabled, if invoking from setup, set the environment variable DN3FFB to $false to prevent this..."
 }
 
 $RepoRoot = "$PSScriptRoot"
@@ -74,7 +85,7 @@ $NoTimestampPackageVersion=$env:PACKAGE_VERSION + "-" + $env:BUILD_QUALITY
 
 $TimestampPackageVersion=$NoTimestampPackageVersion + "-" + [System.DateTime]::Now.ToString("yyyyMMdd") + "-" + $env:BUILD_NUMBER
 
-& dotnet msbuild $RepoRoot\build.proj /p:IsFullFrameworkBuildSupported=true /p:New3RuntimeIdentifier=$Runtime /p:Configuration=$Configuration
+& dotnet msbuild $RepoRoot\build.proj /p:IsFullFrameworkBuildSupported=$PerformFullFrameworkBuild /p:New3RuntimeIdentifier=$Runtime /p:Configuration=$Configuration
 $NewPath = $DevDir + ";" + (($env:PATH.Split(';') | where {-not $_.StartsWith($RepoRoot)}) -join ";")
 [Environment]::SetEnvironmentVariable("Path", "$NewPath", [System.EnvironmentVariableTarget]::User)
 $env:Path = $NewPath

--- a/setup.cmd
+++ b/setup.cmd
@@ -15,12 +15,14 @@ PUSHD %~dp0\src
 IF "%DN3B%" == "" (SET DN3B=Release)
 echo Using build configuration "%DN3B%"...
 
+IF "%DN3FFB%" == "" (SET DN3FFB=$true)
+
 CALL "%~dp0\harderreset.cmd"
 
 mkdir %~dp0\dev 1>nul
 
 echo "Calling build.ps1"
-powershell -NoProfile -NoLogo -Command "& \"%~dp0build.ps1\" -Configuration %DN3B%; exit $LastExitCode;"
+powershell -NoProfile -NoLogo -Command "& \"%~dp0build.ps1\" -Configuration %DN3B% -$PerformFullFrameworkBuild %DN3FFB%; exit $LastExitCode;"
 
 echo Artifacts built and placed.
 

